### DISCUSSION TITLE
Dissonance in Astro's messaging

### DISCUSSION THREAD

**Discussion by ohansemmanuel** [withastro/docs#5186] [Ideas]
Given the growth of the framework and changes over the past months, there's starting to be a dissonance in the messaging on the main landing page (website) and the documentation site. 

Are there plans to consolidate the messaging? 

![CleanShot 2023-10-24 at 05 30 09@2x](https://github.com/withastro/docs/assets/10930234/3f7c612d-c9f6-4e21-9eb0-c9f25bf3de96)

![CleanShot 2023-10-24 at 05 30 34@2x](https://github.com/withastro/docs/assets/10930234/628cbb77-d08a-4d58-9ffe-eda9a28d8431)

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

Since https://docs.astro.build/en/concepts/why-astro/ no longer mentions what's in your screenshot, I guess this is solved and we can close this one.

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
